include/revng/Model/Register.h

```diff
@@ -50,6 +59,7 @@ type: enum
   - name: xmm5_x86_64
   - name: xmm6_x86_64
   - name: xmm7_x86_64
+  - name: fs_x86_64
```

Comment:
This scares me a bit, is there any ABI using `fs`?

Reply:
It's considered callee-preserved in some ABIs.